### ISSUE TITLE
"Recently viewed" section in sidebar loses items

### ISSUE THREAD

**Issue by pgorod** [salesagility/SuiteCRM#5546]
#### Issue
Users are reporting having less items in Recently viewed than usual...

#### Expected Behavior
Recently viewed area should show 5 items.

#### Actual Behavior
I'm not exactly sure what is the behavior, there might be other factors I'm not seeing yet, but it seems to be losing on item every time you go into a List view of a different module.

#### Steps to Reproduce
1. See how many items you have in Recently viewed, on the sidebar.
2. Go into Accounts module list view - you lose one item
3. Go into Contacts module list view - you lose one more item
4. etc., all the way down to zero items shown!

#### Environment
* SuiteCRM 7.10.2
* PHP 7.0
* Ubuntu 16.04

**Comment by edmondklai** [comment]
It is strange, it happens sometimes.  Sometimes it works fine, sometimes it replaces an item, sometimes it loses an item.

**Comment by pgorod** [comment]
Yes, it is erratic. Since this regression comes (I think) from changes made related to Email addresses showing in the sidebar, maybe it is dependent on whether the list shows email addresses or not? Or on whether the record shown has an email? But I don't have time to check right now.

**Comment by pgorod** [comment]
Maybe this isn't such a low priority, it affects useful functionality in every screen of SuiteCRM... I think we should fix these regressions quickly, it's easier to do when not much time has gone by since the change that broke the functionality.

**Comment by brendanbell** [comment]
Hi,

I just did a clean in install of 7.10.3. Straight away I noticed that I cannot create any new user records and the left hand recently viewed items have no entries.

Ubuntu 16.04.4
mySQL
php 7.0

7.10.2 Version
![suite-crm-7 10 2-users](https://user-images.githubusercontent.com/1163263/38450662-140ca57e-3a65-11e8-9dce-ad8e1fc92eed.png)

7.10.3 Version
![suite-crm-7 10 3-users](https://user-images.githubusercontent.com/1163263/38450663-1fb37394-3a65-11e8-8bf3-912affc6ba65.png)

**Comment by brendanbell** [comment]
Just an update.

I completely removed (files and mysqldb) 7.10.3 and re-installed 7.10.2 and the side menu is now showing.

**Comment by pgorod** [comment]
@brendanbell I think your issue is this one instead: #5660

There will be a 7.10.4 release sooner than planned to make sure that one is fixed ASAP.

This issue here is just about `Recently viewed` items, not Actions.

**Comment by johnM2401** [comment]
Hey!
Thank you for raising this!

This appears to have since been resolved.
I've given this a try in the latest release and have not been able to replicate this.

I've tried navigating through the CRM as both Admin & non-admin, in English and German language packs
However, the sidebar seems to remain consistent. 
(In both Recently Viewed & Favourites remaining/updating as appropriate, as well as Sidebar actions being shown as appropriate)

I'm going to close this off for now.

However, if you are still seeing this issue in the latest releases of SuiteCRM, please don't hesitate to get in touch.

Thank you for your contributions to the project!
